### PR TITLE
Acronym reference list

### PR DESCRIPTION
**Pull request by traytrast**

### CODE REVIEW

File: work-in-progress/acronyms.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* `DC`: Debt Ceiling
+* `DC-IAM`: Debt Ceiling Instant Access Module
+* `DOI`: Declaration of Intent
+* `DSR`: Dai Savings Rate
+* `DSS`: Dai Stablecoin System
+* `DUX`: Development & UX (User Experience)
+* `D3M`: Direct Deposit DAI Module
+* `ESM`: Emergency Shutdown Module
+* `FTE`: Full Time Equivalent
+* `GSM`: Government Security Module
```
Comment (patrick-j-govalpha) [inline work-in-progress/acronyms.md:20]: It's *Governance* Security Module, not Government

File: work-in-progress/acronyms.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+* `GDPR`: General Data Protection Regulation
+* `IPFS`: InterPlanetary File System
+* `IS`: Immunefi Security Core Unit
```
Comment (patrick-j-govalpha) [inline work-in-progress/acronyms.md:23]: Include IS-001

File: work-in-progress/acronyms.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* `MEV`: Miner Extracted Value
+* `MKR`: Maker
+* `NFT`: Non-Fungible Tokens
+* `OMC`: Open Markets Committee
+* `LP`: Liquidity Provider, Liquidation Penalty
+* `LR`: Liquidation Ratio
+* `MIP`: Maker Improvement Proposal
+* `OKR`: Objectives and Key Results
+* `OSM`: Oracle Security Module
+* `PE/PECU`: Protocol Engineering
```
Comment (patrick-j-govalpha) [inline work-in-progress/acronyms.md:37]: Add Core Unit suffix and (PE-001)

File: work-in-progress/acronyms.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+* `PSM`: Peg Stability Module
+* `RoA`: Return over Asset
+* `RoE`: Return over Equity
+* `RP`: Risk Premium
+* `RFC`: Request For Comment
+* `RWA`: Real-World Assets
+* `RWF`: Real-World Finance Core Unit
```
Comment (patrick-j-govalpha) [inline work-in-progress/acronyms.md:44]: Add RWF-001

File: work-in-progress/acronyms.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+* `SCD/SAI`: Single Collateral DAI
+* `SES`: Sustainable Ecosystem Scaling Core Unit
```
Comment (patrick-j-govalpha) [inline work-in-progress/acronyms.md:46]: Add SES-001

File: work-in-progress/acronyms.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+* `SB`: Surplus Buffer
```
Comment (patrick-j-govalpha) [inline work-in-progress/acronyms.md:47]: I'd maybe also add "/ System Surplus Buffer"

**Comment by traytrast** [comment]
Okay, I think I addressed all the comments, let me know if there's anything else you see!

**Comment by patrick-j-govalpha** [comment]
LGTM @Gala-G if you're happy can you merge and then I'll make edits on my end

**Comment by Gala-G** [comment]
@patrick-j-govalpha yes! I’ll take a look at it tonight and merge it. Thank you both !

**Pull request merged**